### PR TITLE
Changed instructions for config flow

### PR DESCRIPTION
**Pull request by pattyland**
## Proposed change

The avea integration can be configured via the user interface now so it need new instructions for configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34468
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/avea.markdown [hunk 1]
```diff
@@ -13,12 +13,15 @@ ha_domain: avea
...
-  - platform: avea
-```
+- Go to **Configuration** -> **Integrations**.
+- Click on the `+` in the bottom right corner to add a new integration.
+- Search and select the **Avea** integration form the list.
+- Follow the instruction on screen to add the lights.
+
+<div class='note'>
+If you have previously configured this integration using configuration.yaml, you can delete this entry.
+</div>
```
Comment (frenck) [inline source/_integrations/avea.markdown]: This should be part of the breaking changes section, not (what is going to be) the current documentation.

**Review by frenck — DISMISSED** [review]
Thanks, @pattyland! Left a small comment, could you take a look?

Thanks! 👍

File: source/_integrations/avea.markdown (hunk 1 above)
Comment (pattyland) [reply source/_integrations/avea.markdown]: As I added this also in [my PR for Core](https://github.com/home-assistant/core/pull/34468) I guess this should suffice?

Comment (frenck) [reply source/_integrations/avea.markdown]: Yes, that is enough.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Comment by pattyland** [comment]
@frenck Could you reopen this PR? There is a new pending parent PR but there a no changes necessary for the documentation

**Pull request closed without merging**